test/e2e/create_test.go

```diff
@@ -150,6 +150,10 @@ var _ = Describe("Podman create", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
 
+		session = podmanTest.Podman([]string{"wait", "test"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
```

Comment:
Can you elaborate on why all the waits are needed?

Reply:
I added them because of the inherent race condition or trying to read a log while a container is still running, each one of these containers is detatched, and we could read the log before the container has written the log.